docs/dml-of-carbondata.md

```diff
@@ -447,13 +452,17 @@ CarbonData DML statements are documented here,which includes:
     
 ### DELETE
 
-  This command allows us to delete records from CarbonData table.
+  This command allows us to delete records from CarbonData table. Without providing expression, it will delete all the records from table.
```

Comment:
this can be added after syntax, as a note

Reply:
Done